tests/virt/cluster/longevity_tests/utils.py

```
@@ -42,13 +42,13 @@ def decorate_log(msg):
     return f"{msg_decor}{msg}{msg_decor}"
 
 
-def run_migration_loop(iterations, vms_with_pids, os_type, wsl2_guest=False):
+def run_migration_loop(client, iterations, vms_with_pids, os_type, wsl2_guest=False):
```

Ruff (0.15.14)

[warning] Boolean default positional argument in function definition

(FBT002)